src/neovis.js

```diff
@@ -158,6 +162,8 @@ export default class NeoVis {
 		// node caption
 		if (typeof captionKey === 'function') {
 			node.label = captionKey(neo4jNode);
+		} else if (captionKey && (typeof neo4jNode.properties[captionKey] === 'number' || Neo4j.isInt(neo4jNode.properties[captionKey]))) {
```

Comment:
why `Neo4j.isInt` is not enough?

Reply:
isInt only checks neo4j int type if I am not wrong. To handle float we need to do another number check.

Reply:
Oh, I toght there was a `&&` there, my bad